addons/project/views/project_views.xml

```diff
@@ -856,22 +865,22 @@
                             <div class="o_field_widget o_stat_info">
-                                <span class="o_stat_value"><field name="rating_percentage_satisfaction" widget="integer"/>%</span>
-                                <span class="o_stat_text">Satisfaction</span>
+                                <span class="o_stat_value"><field name="rating_avg" nolabel="1"/> / 5</span>
+                                <span class="o_stat_text">Ratings</span>
                             </div>
                         </button>
-                        <button name="action_open_ratings" type="object" attrs="{'invisible': ['|', '|', ('rating_count', '=', 0), ('rating_percentage_satisfaction', '&gt;=', 66.6), ('rating_percentage_satisfaction', '&lt;', 33.3)]}" class="oe_stat_button" icon="fa-meh-o text-warning" groups="project.group_project_rating">
+                        <button name="action_open_ratings" type="object" attrs="{'invisible': ['|', '|', ('rating_count', '=', 0), ('rating_avg', '&gt;=', 3.6), ('rating_avg', '&lt;', 2.3)]}" class="oe_stat_button" icon="fa-meh-o text-warning" groups="project.group_project_rating">
                             <div class="o_field_widget o_stat_info">
-                                <span class="o_stat_value"><field name="rating_percentage_satisfaction" widget="integer"/>%</span>
-                                <span class="o_stat_text">Satisfaction</span>
+                                <span class="o_stat_value"><field name="rating_avg" nolabel="1"/> / 5</span>
+                                <span class="o_stat_text">Ratings</span>
                             </div>
                         </button>
-                        <button name="action_open_ratings" type="object" attrs="{'invisible': ['|', '|', ('rating_count', '=', 0), ('rating_percentage_satisfaction', '&gt;=', 33.3),  ('rating_percentage_satisfaction', '&lt;', 0)]}" class="oe_stat_button" icon="fa-frown-o text-danger" groups="project.group_project_rating">
+                        <button name="action_open_ratings" type="object" attrs="{'invisible': ['|', ('rating_count', '=', 0),  ('rating_avg', '&gt;=', 2.3)]}" class="oe_stat_button" icon="fa-frown-o text-danger" groups="project.group_project_rating">
                             <div class="o_field_widget o_stat_info">
-                                <span class="o_stat_value"><field name="rating_percentage_satisfaction" widget="integer"/>%</span>
-                                <span class="o_stat_text">Satisfaction</span>
+                                <span class="o_stat_value"><field name="rating_avg" nolabel="1"/> / 5</span>
+                                <span class="o_stat_text">Ratings</span>
                             </div>
```

Comment:
It is not related to your diff, but I am wondering if we couldn't get something lighter like we have in Helpdesk? I remembered that I recently had to tweak things here in order to ensure that the design is both nice in community and enterprise. We should ensure that what I propose does not break that.
I thought of a structure like this:
```xml
<button name="...>
    <i class="fa fa-fw o_button_icon fa-smile-o text-success" attrs=".../>
    <i class="fa fa-fw o_button_icon fa-meh-o text-warning" attrs=".../>
    <i class="fa fa-fw o_button_icon fa-frown-o text-danger" attrs=".../>
    <div class="o_field_widget o_stat_info">
  ```